Makefile

```diff
@@ -33,7 +33,7 @@ examples:
 
 .PHONY: test tests
 test tests:
-	ginkgo run -r
+	ginkgo run -r .
```

Comment:
@jhernand PTAL at this specifically.   there was a recent change here.

Reply:
I think this doesn't need to be changed. Why do you think it does?

Reply:
I will assume that you think this change is needed to make the tests pass, but it isn't. Some of the tests fail randomly because they are time sensitive and timing in the GitHub actions environment seems to be very different to our development environments. I am trying to find a solution for that.

I am closing this and replacing it with another pull request to include other changes that have been made to the model.